lib/image_db.rb

```diff
@@ -67,7 +77,7 @@ def full_update
     # NOTE: Offset count verified 2015-07-22
     (0..6000).step(60) do |offset|
       rss_url = "http://backend.deviantart.com/rss.xml?q=favby%3Acalvin166%2F1422412&type=deviation&offset=#{offset}"
-      p ({ :deviant => "calvin166", :offset => offset })
+      p ({ deviant: "calvin166", offset: offset })
```

Comment:
`(...)` interpreted as grouped expression.<br>Do not write to stdout. Use Rails' logger if you want to log.